bin/command.php

```diff
@@ -335,6 +335,31 @@ private static function gen_cmd_pages( $cmd, $parent = array() ) {
 			$docs = preg_replace( '/^(\[?)--/m', '\1\--', $docs );
 			$docs = preg_replace( '/^\s\s--/m', '  \1\--', $docs );
 
+			// Remove wordwrapping from docs
+			// Match words, '().,;', and --arg before/after the newline
+			$bits = explode( "\n", $docs );
+			$in_yaml_doc = false;
+			for ( $i=0; $i < count( $bits ); $i++ ) {
```

Comment:
CS: spaces around assignment

Reply:
PHPCS isn't enforced in this repo

Reply:
Wouldn't there be merit in being "consistently" consistent? :)

Reply:
Not until it's enforced through CI. CS feedback should only happen through CI. If it's happening in a pull request, then it's not an enforceable standard.